setup.cfg

```diff
@@ -6,7 +6,12 @@ astropy_header = true
 #doctest_plus = disabled
 #text_file_format = rst
 xfail_strict = true
-addopts = -p no:warnings
+filterwarnings =
+    error
+    ignore:numpy.ufunc size changed:RuntimeWarning
+    ignore:numpy.ndarray size changed:RuntimeWarning
+    ignore::DeprecationWarning:ginga
+    ignore:Marker set named:UserWarning
```

Comment:
Ignoring this to avoid conflict with #126 . I will open a follow up issue to undo this in the future.